SDK_dotnet/Clients/GluwaClient.cs

```diff
@@ -279,7 +288,7 @@ public sealed class GluwaClient
 
             BigInteger convertAmount = GluwacoinConverter.ConvertToGluwacoinBigInteger(amount);
             BigInteger convertFee = GluwacoinConverter.ConvertToGluwacoinBigInteger(getFee.Data.MinimumFee.ToString());
-            int nonce = ((int)(DateTime.UtcNow - new DateTime(1970, 1, 1)).TotalSeconds);
+            nonce = ((int)(DateTime.UtcNow - new DateTime(1970, 1, 1)).TotalSeconds).ToString();
```

Comment:
so what happens if I pass in null to this method? it will override that value? seems wrong